MediaManager/Platforms/Apple/Notifications/NotificationManager.cs

```diff
@@ -7,6 +7,11 @@ namespace MediaManager.Platforms.Apple.Notifications
 {
     public class NotificationManager : NotificationManagerBase
     {
+        public NotificationManager()
```

Comment:
Maybe move this to the base class, and set all properties in here, so all platforms get all triggered?

Reply:
I wasn't sure about that but if you have also same thinking ;)

Reply:
done. ;)